build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -96,6 +96,9 @@ fun configureSourcesVariant() {
         main.groovy.srcDirs.forEach {
             outgoing.artifact(it)
         }
+        main.resources.srcDirs.forEach {
+            outgoing.artifact(it)
+        }
```

Comment:
This doesn't seem quite right.  I don't think we have this, but if we had source files in the resources directory, this would cause us to consider these part of our sources too.